util/logger/src/lib.rs

```diff
@@ -49,3 +51,7 @@ use lazy_static::lazy_static;
 lazy_static! {
     pub static ref slogger: StructuredLogger = StructuredLogger::create();
 }
+
+lazy_static! {
+    pub static ref metric_logger: Metric = Metric::new();
```

Comment:
How about passing `Metric` to the modules which use it instead of declaring global one.

Reply:
Are you saying about passing the metric variable as a function argument?
Then calculating the metric of very nested code became very difficult.